frontend/src/components/login_page/login.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react'
+import './login.css'
+import Logo_orange_tan from '../assets/logo_orange_tan.svg'
+import Logo_orange from '../assets/logo_orange_black.svg'
+
+class Form extends Component {
```

Comment:
really close! let's shy away from using these older style "Class component" definitions. Modern react is written slightly differently 

may be helpful: https://www.twilio.com/en-us/blog/react-choose-functional-components